lib/json_dao_container.es6.js

```diff
@@ -24,23 +65,24 @@ foam.CLASS({
     'foam.core.StubFactorySingleton',
     'foam.dao.ClientDAO',
     'foam.dao.ReadOnlyDAO',
+    'org.chromium.apis.web.JsonDAOContainerConfig',
+    'org.chromium.apis.web.JsonDAOContainerMode',
     'org.chromium.apis.web.SerializableHttpJsonDAO',
+    'org.chromium.apis.web.SerializableLocalJsonDAO',
   ],
   imports: ['info'],
   exports: ['as container'],
 
   properties: [
     {
-      class: 'Function',
-      name: 'getJsonUrlForClass',
-      factory: function() {
-        return cls => `https://storage.googleapis.com/web-api-confluence-data-cache/latest/json/${cls.id}.json`;
-      },
-    },
-    {
-      name: 'stubFactory',
+      class: 'FObjectProperty',
+      of: 'org.chromium.apis.web.DAOContainerConfig',
+      name: 'config',
       factory: function() {
-        return this.StubFactorySingleton.create();
+        return this.DAOContainerConfig.create({
```

Comment:
I'm not sure that this container is providing much value over embedding the fields on the class directly.

Reply:
Removed the Config class.